src/test/java/com/sap/sgs/phosphor/fosstars/model/value/VulnerabilitiesValueTest.java

```diff
@@ -14,8 +15,10 @@
   public void serialization() throws IOException {
     ObjectMapper mapper = new ObjectMapper();
     VulnerabilitiesValue vulnerabilityValue =
-        new VulnerabilitiesValue(new Vulnerabilities(new Vulnerability("https://bugtracker/1"),
-            new Vulnerability("https://bugtracker/2")));
+        new VulnerabilitiesValue(
+            new Vulnerabilities(
```

Comment:
If `VulnerabilitiesValue` itself contains list of `Vulnerabilities`, then do we need to do a wrapper here?

Reply:
`VulnerabilitiesValue` contains an instance of `Vulnerabilities` which is a collection of vulnerabilities. In fact, it doesn't use a list, it uses a set. I know that `Vulnerabilities` may look redundant but it also provide multiple convenient methods to work with a collection of vulnerabilities. Of course, we can add a constructor `VulnerabilitiesValue(List ... )` if necessary.